features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/js/edit.asset.js

```diff
@@ -86,7 +86,7 @@ $(function() {
 			},
 			success : function(response) {
 
-				var result = JSON.parse(response);
+				var result = response;
```

Comment:
removed redundant variable 'result' in commit 89598a792a3802f2625a5ce84fb9e041b3e51e04 of PR #240